test/fixtures/benchmarks.yml

```diff
@@ -1,5 +1,6 @@
 # Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
-
+_fixture:
+  model_class: Xccdf::Benchmark
```

Comment:
This is set here now globally, so we don't need any more `fixture_class` declarations.